src/app/login/page.tsx

```
@@ -0,0 +1,52 @@
+  const [password, setPassword] = useState("");
+  const router = useRouter();
+
+  const handleLogin = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post("/api/login", { username, password });
+      if (response.data.success) {
+        localStorage.setItem("token", response.data.token);
+        router.push("/");
+      }
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)